formstack-consents/project/plugins.sbt

```diff
@@ -1 +1,3 @@
-addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.0.0")
\ No newline at end of file
+addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.0.0")
+
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
```

Comment:
Any reason why you used version `0.10.0-RC1` elsewhere but `0.9.2` here?

Reply:
Ah sorry. Copy pasted from payment-failure plugins.sbt which was out of date. now fixed.